src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit POISONED_F , [hl]
+	ret z
+	push hl
+	bit DOUBLE_POISONED_F, [hl]
+	ld a, PSN_DAMAGE
+	ldtx hl, Received10DamageDueToPoisonText    ; copies from the byte of the damage due to poison by 10
```

Comment:
Now that you are properly using the text label, you can delete the comment since it doesn't add more information (likewise for the comment below for the other text label).